pygments/lexers/modeling.py

```diff
@@ -363,3 +364,66 @@ def analyse_text(text):
             return 1.0
         else:
             return 0.0
+            
+class UmpleLexer(JavaLexer):
+  """
+    Pygments lexer for Umple <https://cruise.umple.org/umple/> source code
+
+    .. versionadded::1.31 
+  """
+
+
+  name="Umple"
+  aliases=['umple']
+  filenames = ['*.ump', '*.umpt']
+  
+  tokens = {
+    'root': [
+      (r'(active|after|all|around|atomic|attr|attribute|before|'
+      r'custom|deactivate|during|emit|entry|execute|exit|generate|generated|hops|'
+      r'include|includeFilter|inner|isA|key|model|[oO]n|[oO]ff|pooled|position|post|prefix|queued|regex|singleton|'
+      r'sorted|sub|subclass|suffix|super|superclass|top|trace|unspecified)\b',
+       Keyword),
+      # method names
+      (r'((?:(?:[^\W\d]|\$)[\w.\[\]$<>]*\s+)+?)'  # return arguments
+      r'((?:[^\W\d]|\$)[\w$]*)'                  # method name
+      r'(\s*)(\()',                              # signature start
+      bygroups(using(this), Name.Function, Text, Punctuation)),
+      (r'(activate|afterEvery|around_proceed)',Name.Function),
+      (r'(autounique|const|defaulted|external|filter|[fF]inal|fixml|generic|immutable|internal|ivar|lazy|mixset|'
+      r'pre|settable|statemachine|test|tracer|trait|unique)\b',
+       Keyword.Declaration),
+      (r'((queued|pooled)?sm)\b', Keyword),
+      (r'(association|associationClass)\b', Keyword.Declaration, 'associations'),
+      (r'(distributable)\b(\s+)(RMI|WS|off|forced)', bygroups(Keyword.Reserved, Text, Keyword.Reserved)),
+      (r'(depend)(\s+)([\w.]+\*?)', bygroups(Keyword.Reserved, Text, Keyword.Namespace)),
+      (r'(namespace)\b(\s+)([\w.]+\*?|-)(\s+)?(--redefine)?',
+       bygroups(Keyword.Namespace, Text, Name.Namespace, Text, Keyword)),
```

Comment:
Here and in other places: We're trying to be stricter about `Text` and `Text.Whitespace`. Can you please change the token type to `Text.Whitespace` for the `\s` class?

Reply:
This change has been made too